.github/workflows/rector.yaml

```diff
@@ -72,4 +72,3 @@ jobs:
                 with:
                     commit_message: [ci-review] Rector Rectify
                     commit_author: GitHub Action <actions@github.com>
-                    commit_user_email: action@github.com
```

Comment:
These values might need quoting, as they contain weird characters

Reply:
It works 🎉 https://github.com/rectorphp/rector-src/pull/3426/commits/4fde336cf238bc44373c5fab2fbcb13be095c58b